utilities/cli/holohub.py

```diff
@@ -177,6 +178,11 @@ def _create_parser(self) -> argparse.ArgumentParser:
             choices=["debug", "release", "rel-debug"],
             help="Build type (debug, release, rel-debug)",
         )
+        build.add_argument(
+            "--with",
```

Comment:
Based on feedback in today's internal discussion

```suggestion
            "--build_with",
```